### PR TITLE
Simplified ColumnName handling, moved release-related plugins to separate profile. Added new module for ChromaTOF CSV file support.

### PR DESCRIPTION
**Pull request by nilshoffmann**

### CODE REVIEW

**Comment by tomas-pluskal** [comment]
THanks @nilshoffmann ! Looks good to me.
Thomas @dyrlund , can you check the ChromaTOF CSF module, if you have any comments to it?

File: msdk-datamodel/src/main/java/io/github/msdk/datamodel/featuretables/ColumnName.java [hunk 1]
```diff
@@ -28,7 +26,7 @@
     GROUPID("Group ID", Integer.class),
     MZ("m/z", Double.class),
     PPM("ppm", Double.class),
-    RT("RT", ChromatographyInfo.class),
+    RT("RT", Double.class),
     RTSTART("RT Start", Double.class),
```
Comment (dyrlund) [inline msdk-datamodel/src/main/java/io/github/msdk/datamodel/featuretables/ColumnName.java]: Why is the class of RT changed to Double from ChromatographyInfo?

Comment (tomas-pluskal) [inline msdk-datamodel/src/main/java/io/github/msdk/datamodel/featuretables/ColumnName.java]: Hm, should be ChromatographyInfo, I think.

**Comment by dyrlund** [comment]
@tomas-pluskal I'm not sure I understand your comment about checking the ChromatTOF CSF module. The import module is still under development by Nils.

**Comment by tomas-pluskal** [comment]
Yes, sorry, I wrote that before I actually checked the code :)

File: msdk-datamodel/src/main/java/io/github/msdk/datamodel/featuretables/ColumnName.java (hunk 1 above)
Comment (nilshoffmann) [inline msdk-datamodel/src/main/java/io/github/msdk/datamodel/featuretables/ColumnName.java]: Thanks for spotting this, I will check where I need to fix this. There was an issue with one of the tests, when it was ChromatographyInfo. Maybe on of the builders was not setting the type correctly. Will fix this with the next pull request.

**Comment by nilshoffmann** [comment]
Yes, I only added it as a stub for now. More will follow soon.

**Pull request merged**